perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -678,7 +681,7 @@ def Run(self, vm, metadata):
       the results of all scenarios run here.
 
     """
-    metadata['azure_lib_version'] = _GetAzureClientLibVersion(vm)
+    metadata['azure_lib_version'] = _GetClientLibVersion(vm, 'azure')
```

Comment:
You used a constant for BOTO_LIB_VERSION but not for 'azure_lib_version'. Is that  because 'azure_lib_version' is only used once?